src/database/services/keycloak-api-service.ts

```diff
@@ -36,7 +61,11 @@ export const CreateKeycloakApiService = (): KeycloakApiService => {
         );
       }
 
-      return response.json();
+      const users: KeycloakProfile[] = await response.json();
+      return users.map((user) => ({
+        ...user,
+        severaGuid: (user as CustomKeycloakProfile).severaGuid ?? "Users SeveraGuid not found in Keycloak",
```

Comment:
We don't want to set the severaGuid as a error message string. Let's set it as undefined or null- you decide.
Also Stepan suggested last week we change all references to Guid to Id and I tend to agree. Lets do this here unless you disagree?

Reply:
Stepan success that we use severaUserGuid. I think that would be fine.